web/src/compose_ui.ts

```diff
@@ -348,27 +381,33 @@ export function handle_keydown(event, $textarea) {
     const isCmdOrCtrl = common.has_mac_keyboard() ? event.metaKey : event.ctrlKey;
 
     if (type && isCmdOrCtrl) {
-        format_text($textarea, type);
+        format_text($textarea, type, "");
```

Comment:
I don't think this is a necessary API change; just set a default in the `format_text` definition itself.